packages/material-ui-lab/src/ClockPicker/ClockNumbers.tsx

```diff
@@ -75,7 +76,7 @@ export const getMinutesNumbers = ({
   value,
   isDisabled,
   getClockNumberText,
-}: Omit<GetHourNumbersOptions, 'ampm' | 'date'> & { value: number }) => {
+}: DistributiveOmit<GetHourNumbersOptions, 'ampm' | 'date'> & { value: number }) => {
```

Comment:
We were using the built-in Omit helper previously. Why change it for the distributive one? I guess it's the same question for all the other cases. When should we use one over the other?

Reply:
I went through all occurances and applied the following:

- If applied to an interface, use the built-in `Omit`, since the distributive one has no effect;
- If applied to a union type or to a generic parameter, use `DistributiveOmit`.